src/main/java/nl/rijksoverheid/mev/gezagsmodule/domain/gezagvraag/OuderOverledenOfOnbevoegdTotGezag.java

```diff
@@ -18,7 +22,17 @@ public class OuderOverledenOfOnbevoegdTotGezag extends GezagVraag {
     private static final String V4A_3_NEE_OUDER1 = "Nee_ouder1";
     private static final String V4A_3_NEE_OUDER2 = "Nee_ouder2";
     private static final String V4A_3_NEE = "Nee";
-    private static final String V4A_3_JA = "Ja";
+    private static final Map<String, String> JA_ANTWOORDEN = Map.of(
+        "?c", "Ja_ouder_onder_curatele",
+        "?m", "Ja_ouder_minderjarig",
+        "?o", "Ja_ouder_overleden",
+        "cc", "Ja_beiden_onder_curatele",
+        "cm", "Ja_ouder_onder_curatele_en_andere_ouder_minderjarig",
```

Comment:
in deze rule is er altijd maar 1 ouder ik zou daarom ook alleen 1 ouder uitkomsten verwachten

Reply:
Besproken in https://github.com/BRP-API/brp-api-gezag/pull/227/files#r1890171527.